modules/popup-stack/README.md

```diff
@@ -115,6 +116,13 @@ export interface PopupStackItem {
 }
 ```
 
+### createContainer
+
+Create a HTMLElement as the container for the popup stack item. The returned element reference will
+be the reference to be passed to all other methods. The Popup Stack will control when this element
+is added and removed from the DOM as well as the `z-index` style property. Your content should be
+added to this element.
```

Comment:
Should there be a mention of `createAdapter` in the Popup Stack README as well?

Reply:
Yes